e2e/tenv.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable import/no-nodejs-modules */
+import childProcess from 'child_process';
+import fs from 'fs/promises';
+import { createServer } from 'http';
+import { AddressInfo } from 'net';
+import path from 'path';
+import { setTimeout } from 'timers/promises';
+
+let leftovers: Proc[] = [];
+afterAll(async () => {
+  await Promise.allSettled(
+    leftovers.map(proc => {
+      proc.kill();
+      return proc.waitForExit;
+    }),
+  );
+  leftovers = [];
+});
```

Comment:
I didn't even realized you could use this function in a utility file outside of the test itself. This is very cool